wagtail/admin/views/account.py

```diff
@@ -138,10 +138,18 @@ def render(self):
 
 class NameEmailSettingsPanel(BaseSettingsPanel):
     name = "name_email"
-    title = gettext_lazy("Name and Email")
+    # title = gettext_lazy("Name and Email")
     order = 100
     form_class = NameEmailForm
 
+    @property
```

Comment:
```suggestion
    @cached_property
```

Probably best to use cached property here, changing settings requires a restart and it avoids us checking this each view load. 

https://docs.python.org/3/faq/programming.html#faq-cache-method-calls